packages/actors_api/README.md

```diff
@@ -0,0 +1,47 @@
+[![](https://user-images.githubusercontent.com/1248214/57789522-600fcc00-7739-11e9-86d9-73d7032f40fc.png)
+](https://kilt.io)
+
+![Lint and Test](https://github.com/KILTprotocol/sdk-js/workflows/Lint%20and%20Test/badge.svg)
+
+Data sovereignty and interoperability
+
+# Kilt SDK Actors API
+
+This package groups KILT's functionality according to the three most common roles a KILT user may assume:
+
+- Claimer
+- Attester
+- Verifier
+
+It is meant to make relevant functionality easily accessible and discoverable.
+
+## Claimer
+
+This section contains functions to request an attestation and to send it to a Verifier (presentation).
+`requestAttestation` allows you to request an attestation for a claim from an Attester, which returns a message object that can sent off to the Attester and a session object to keep track of the details of the request.
```

Comment:
To make this a new paragraph in the resulting html, you have to either add 2 spaces to the end of the previous line for a single carriage return or a new line for 2 carriage returns.

Reply:
pro tip: newline after each sentence. Is ignored, as you say, but makes diffs so much cleaner and more useful.
Trust me, I wrote my thesis in markdown :D

Reply:
but I agree that paragraphs would be nice here